pkg/apis/fpga.intel.com/v1/types.go

```diff
@@ -18,7 +18,9 @@ type AcceleratorFunction struct {
 
 // AcceleratorFunctionSpec contains actual specs for AcceleratorFunction
 type AcceleratorFunctionSpec struct {
-	AfuID string `json:"afuId"`
+	AfuID       string `json:"afuId"`
+	InterfaceID string `json:"interfaceId"`
+	Mode        string `json:"mode"`
 }
```

Comment:
I suppose in this case we'd need to maintain two versions of the webhook: "mode-ful" and "mode-less". Making the webhook modeless is already a breaking change.

Reply:
> Making the webhook modeless is already a breaking change.

I was thinking v2 because of the breakage but did not think of the need to have both versions maintained (beyond what we have in the release branches).

Reply:
Right. That makes sense if there are installed old mappings in a cluster. Will update the version on Monday.

Reply:
@kad @bart0sh your thoughts?

Reply:
I moved the fpga.intel.com API group to v2. It increased the size of the patch, but all tests pass.